### PR TITLE
[NUI.Scene3D] Transition animation with FieldOfView

### PR DESCRIPTION
**Pull request by hinohie**
Add camera's FieldOfView animated by
Scene3D.SceneView's CameraTransition API.

Signed-off-by: Eunki, Hong <eunkiki.hong@samsung.com>

### CODE REVIEW

File: src/Tizen.NUI.Scene3D/src/public/Controls/SceneView.cs [hunk 1]
```diff
@@ -287,7 +287,7 @@ public void SelectCamera(string name)
 
         /// <summary>
         /// Starts camera transition from currently selected camera to a camera of index.
-        /// Camera Position and Orientation is smoothly animated.
+        /// Camera Position and Orientation and FieldOfView is smoothly animated.
```
Comment (bshsqa) [inline src/Tizen.NUI.Scene3D/src/public/Controls/SceneView.cs:290]: Camera Position, Orientation and FieldOfView are smoothly animated.

**Review by bshsqa — APPROVED** [review]
I approved this patch but please fix the minor comments.

File: src/Tizen.NUI.Scene3D/src/public/Controls/SceneView.cs (hunk 1 above)
Comment (hinohie) [reply src/Tizen.NUI.Scene3D/src/public/Controls/SceneView.cs]: Thank you. fix done.

File: src/Tizen.NUI.Scene3D/src/public/Controls/SceneView.cs [hunk 2]
```diff
@@ -311,7 +311,7 @@ public void CameraTransition(uint index, int durationMilliSeconds, AlphaFunction
 
         /// <summary>
         /// Starts camera transition from currently selected camera to a camera of input name.
-        /// Camera Position and Orientation is smoothly animated.
+        /// Camera Position and Orientation and FieldOfView is smoothly animated.
```
[the remark this answers is not in the document]
Comment (hinohie) [reply src/Tizen.NUI.Scene3D/src/public/Controls/SceneView.cs]: Thank you. fix done.

**Pull request merged**